ec_subscriber/entrypoint.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+ls
+pwd
```

Comment:
what's the purpose of running ls and pwd in this script?

Reply:
Woopse it was for debugging. They have been removed!